README.md

```diff
@@ -146,7 +182,35 @@ class SomeController {
 
 ### Directive
 
-Static class members of directive controller are used as config directive config.
+You can configure your directive with annotation :
+
+```typescript
+@directive('ngModuleName', {
+    selector:     'atSomeDirective',                //required
+    controllerAs: 'someDirectiveCtrl',              //optional
+    templateUrl:  '/partials/some-directive.html'
+})
+class SomeDirectiveController {
+
+    public static link: angular.IDirectiveLinkFn = (scope, element, attrs, ctrl: SomeDirectiveController) => {
```

Comment:
I think work fine because have these instructions (line 116 - 125)  : 

```javascript 

config = <IDirectiveProperties> angular.copy(directiveSettings);

// Store FuncName as ControllerAs
config.controllerAs = getFuncName(target);
config.controller = target;

angular.forEach(directiveProperties, function(property: string): any {
    if (angular.isDefined(target[property])) {
       config[property] = target[property];
    }
})
```
Link method will be stored in config object.